src/OpenTelemetry.ResourceDetectors.Azure/README.md

```diff
@@ -31,3 +31,53 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
                             .AddDetector(new AppServiceResourceDetector()))
                         .Build();
 ```
+
+The resource detector will record the following attributes:
+
+|                         |
```

Comment:
nit:

This renders a table with an empty header.

We could probably get rid of the line above which says: "The resource detector will record the following attributes:" and add some text as the table header instead. Something like "Attributes recorded by AppServiceResourceDetector".

Reply:
Done, thanks!
still going back-and-forth with Vishwesh to decide the best way to display this information :)